### PR TITLE
Fix make help cmds absence

### PR DESCRIPTION
**Pull request by ggaaooppeenngg**

### CODE REVIEW

**Comment by spxtr** [comment]
Mind suggesting a good reviewer for this?

File: hack/make-rules/make-help.sh [hunk 1]
```diff
@@ -24,8 +24,6 @@ readonly reset=$(tput sgr0)
 KUBE_ROOT=$(dirname "${BASH_SOURCE}")/../..
 ALL_TARGETS=$(make -C "${KUBE_ROOT}" PRINT_HELP=y -rpn | sed -n -e '/^$/ { n ; /^[^ .#][^ ]*:/ { s/:.*$// ; p ; } ; }' | sort)
 CMD_TARGETS=$(ls -l "${KUBE_ROOT}/cmd" |awk '/^d/ {print $NF}')
-PLUGIN_CMD_TARGETS=$(ls -l "${KUBE_ROOT}/plugin/cmd" |awk '/^d/ {print $NF}')
-FED_CMD_TARGETS=$(ls -l "${KUBE_ROOT}/federation/cmd" |awk '/^d/ {print $NF}')
```
Comment (irfanurrehman) [inline hack/make-rules/make-help.sh]: you don't need to remove this, change it to `"${KUBE_ROOT}/cmd"`

File: hack/make-rules/make-help.sh [hunk 2]
```diff
@@ -47,36 +45,6 @@ for tar in $ALL_TARGETS; do
...
-			echo -e "${red}${PLUGIN_CMD_TARGETS}${reset}"
-			make -C "${KUBE_ROOT}" $tar PRINT_HELP=y
-			echo "---------------------------------------------------------------------------------"
-
-			PLUGIN_CMD_FLAG=true
-			continue 2
-		fi
-	done
-
-	for fedcmdtar in $FED_CMD_TARGETS; do
```
Comment (irfanurrehman) [inline hack/make-rules/make-help.sh]: as above, you would not need to remove this portion of code.

**Comment by irfanurrehman** [comment]
@ggaaooppeenngg thanks for the PR. Please update as per comments.
/ok-to-test

File: hack/make-rules/make-help.sh (hunk 1 above)
Comment (irfanurrehman) [reply hack/make-rules/make-help.sh]: yes, that's correct.

**Pull request merged**